deps.edn

```diff
@@ -1,2 +1,4 @@
-{:deps  {org.clojure/tools.cli {:mvn/version "0.3.7"}}
- :paths ["src" "resources"]}
+{:paths ["src" "resources"]
+ :deps  {org.clojure/tools.cli {:mvn/version "0.3.7"}
+         figwheel-sidecar      {:mvn/version "0.5.16-SNAPSHOT"
```

Comment:
Any reason to use a snapshot?

Reply:
Yes. This is required for new version of clojurescript. I did the same here https://github.com/status-im/status-react/pull/4064/files

Reply:
Non snapshot version is out: https://clojars.org/figwheel-sidecar/versions/0.5.16

Reply:
Will fix it.